tools/extras/install_srilm.sh

```diff
@@ -14,7 +14,7 @@ if [ ! -d liblbfgs-1.10 ]; then
 fi
 
 # http://www.speech.sri.com/projects/srilm/download.html
-if [ ! -f srilm.tgz ]; then
+if [ ! -f srilm-1.7.2.tar ]; then
```

Comment:
@naxingyu , I won't comment on the recipe changes per se (not familiar with that)
But I'm not fond of the changes here.
a) what srilm pages do offer for download if you go there is srilm-1.7.2.tar.gz, i.e. I'm not sure where the tar file comes from, but that script won't work for other people now.
b) I'm still in favor of asking users to do a symlink or copy the file as srilm.tgz.
I think it makes sense as we cannot download the file automatically -- so people can use whatever version they have available and do not have to undergo the downloading process every time.  Which I realize it's not true now as I'm thinking about it, as we need 1.7.1 at least, because of the maxent LMs. So I don't really object to having a fixed version and bump it now and then, but I think it's less convenient for us.
c) "$SRILM" is preferred, IMO.
 It's because of cases where the users have space (white character) in the path somewhere. Or perhaps not, but is there any rationale behind changing it?